plugins/pencilblue/controllers/actions/salesforce/logout.js

```diff
@@ -41,6 +42,7 @@ module.exports = function LogOutSFSSOControllerModule(pb) {
                 pb.log.error('Something went wrong during the removal of the cookie : ', e);
             }
 
+            let response = await salesforceStrategyService.logout(this.req);
```

Comment:
This triggers an exception 
![image](https://user-images.githubusercontent.com/16479465/57165608-d9f89a80-6db4-11e9-9740-c52611393779.png)

Interestingly, it seems like the logout issue in SF disappears (and the session destruction works); but when you try to log in again, SF issue appears - again.

I don't think this piece of code is the responsible for the session not being destroyed, and in any case it cannot be placed after the session destruction (it causes the exception I mention)